### PR TITLE
NOD | Increase code coverage

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > Work involves increasing code coverage for our Notice of Disagreement form. Some other changes included fixes to our other Decision Review apps to increase code coverage and use the same tests throughout.
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#71500](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71500)

## Testing done

- _Describe what the old behavior was prior to the change_
  > Missing code coverage
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

|  |  |
| ------- | ----- |
| Before  | before coverage: 96.17% Statements, 86.25% Branches, 90.12% Functions, 96.09% Lines |
| After | after coverage: 100% Statements, 96.36% Branches, 100% Functions, 100% Lines |

## What areas of the site does it impact?

Decision Reviews unit test coverage

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/appeals/996/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -36,7 +36,8 @@ export const ConfirmationPage = () => {
 
   const { submission, data } = form;
   const issues = data ? getIssuesListItems(data) : [];
-  const fullName = `${name.first} ${name.middle || ''} ${name.last}`;
+  const fullName = `${name.first || ''} ${name.middle || ''} ${name.last ||
```
Comment (anniebtran) [inline src/applications/appeals/996/containers/ConfirmationPage.jsx:39]: doesn't need to be covered by this PR but seems like a good candidate for refactoring/a shared function 😄

File: src/applications/appeals/10182/tests/utils/upload.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        </ul>
+      </div>,
+    );
+
+    focusFileCard('test-file.pdf', container);
+    expect(document.activeElement).to.eq($('li:last-child', container));
+  });
+
+  describe('parseResponse', () => {
+    it('should return undefined', () => {
```
Comment (anniebtran) [inline src/applications/appeals/10182/tests/utils/upload.unit.spec.js:25]: does this return `undefined` for a particular value? seems like it's returning an object with some file data in it

File: src/applications/appeals/996/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/appeals/996/containers/ConfirmationPage.jsx:39]: I wish this function was in platform utility code somewhere...

File: src/applications/appeals/10182/tests/utils/upload.unit.spec.js (hunk 2 above)
Comment (Mottie) [reply src/applications/appeals/10182/tests/utils/upload.unit.spec.js:25]: It looks like I mis-labeled that unit test!

File: src/applications/appeals/996/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/appeals/996/containers/ConfirmationPage.jsx]: So, I ended up adding a shared utility function to get the full name & render it.

**Review by anniebtran — APPROVED** [review]
👏

**Review by micahchiang — APPROVED** [review]

**Pull request merged**